parquet-common/src/main/java/org/apache/parquet/ParquetRuntimeException.java

```diff
@@ -43,4 +46,18 @@ public ParquetRuntimeException(Throwable cause) {
     super(cause);
   }
 
+  /**
+   * Call the #close() method on a {@see Closable}, wrapping any IOException
+   * in a runtime exception.
+   *
+   * @param closeable - resource to close
+   */
+  public static void closeQuietly(Closeable closeable) {
+    try {
+      closeable.close();
+    }catch(IOException e){
+      throw new ParquetRuntimeException("Error closing I/O related resources.", e) {};
```

Comment:
I was thinking the method could be used by other consumers, so it might not just be an error closing a stream.